src/components/Book.js

```diff
@@ -12,10 +12,34 @@ const Book = ({ book }) => {
 
   return (
     <div>
-      <li>
-        <p>{book.title}</p>
-        <p>{book.category}</p>
-        <button onClick={() => removeBookFromStore(book.item_id)} type="button">Remove</button>
+      <li className="list-group-item d-flex justify-content-between align-items-start p-3 m-3">
+        <div className="ms-2">
+          <small><i>{book.category}</i></small>
+          <div className="fw-bold text-black m-3">
+            <h3>{book.title}</h3>
+          </div>
+          <button type="button" className="btn btn-link ">Comments</button>
+          <button onClick={() => removeBookFromStore(book.item_id)} type="button" className="btn btn-link ">
+            Remove
+          </button>
+          <button type="button" className="btn btn-link ">Edit</button>
+        </div>
+
+        <div className="flexy mt-2">
+          <div className="oval" />
+          <div>
+            <span className="percent">64%</span>
+            <p className="completed">Completed</p>
+          </div>
+        </div>
+        <div className="d-flex row details">
+          <span className="col currentChapter">Current Chapter</span>
+          <span className="chapter currentLesson "> 176 </span>
+          <div className="col">
+            <button type="submit" className="bg-primary update-btn"> Update Progress</button>
+          </div>
+        </div>
+
```

Comment:
- [ ] Horizontal positioning of each `Book Category` and `Book Title` is a bit off. Also, the white space between the `progress` and `Chapter info` is a bit large, kindly update your `book` component to match the Zeplin design.

Zeplin design
![image](https://user-images.githubusercontent.com/33767951/141312060-725c5507-06cd-4ab9-becc-961e4357b83a.png)

- [ ] Each `book` is missing the `Author` name. Also, the vertical separators `|` between `Comment`, `Remove`, and `Edit` are missing, Kindly update your `book` component to match the Zeplin design(text colors are not matching).
Zeplin design
![image](https://user-images.githubusercontent.com/33767951/141311234-629b724e-69cc-438a-8d31-164a8ed01003.png)